package login.proto;

message AuthSessionTokenReq {
 required bytes session_token = 1;
}

